packages/ts/fusion-frontend/src/Authentication.ts

```diff
@@ -47,6 +50,23 @@ async function doLogout(logoutUrl: string, headers: Record<string, string>) {
   await updateCsrfTokensBasedOnResponse(response);
 }
 
+/**
+ * Remove trailing '/' if the path is not exactly '/'.
+ * Exported for testing purposes only.
+ * @internal
+ */
+export function _removeTrailingSlashFromPath(path: string) {
```

Comment:
Would be nicer to move into a separate `.ts` module and have a separate test suite, that way we could avoid making it a public API.

Reply:
Oh is this only used for cookies, so could be merged with `CookieUtils.ts` perhaps?

Reply:
Yeah good idea, then it won't be in any of the exported entry points. I didn't think of that when adding this.

The logic here is not really specific to cookies in any way but it's currently only used related cookies so I guess it's ok to have it in the CookieUtils for now to avoid creating a new util file just for this. I'll move it.

Reply:
Updated.